src/components/List/List.tsx

```diff
@@ -59,24 +66,27 @@ class List extends UIComponent<Extendable<ListProps>, ListState> {
     accessibility: PropTypes.func,
     debug: PropTypes.bool,
     items: customPropTypes.collectionShorthand,
-    selection: PropTypes.bool,
+    selectable: PropTypes.bool,
     truncateContent: PropTypes.bool,
     truncateHeader: PropTypes.bool,
+    selectedIndex: PropTypes.number,
```

Comment:
if `selectedIndex` is autocontrolled prop, we also need `onSelectedIndexChange` event

Reply:
don't see such pattern  in ```MenuItem```